src/components/history/messages/regular/task/Task.js

```diff
@@ -0,0 +1,124 @@
+import React, {PureComponent, PropTypes} from 'react'
+import {findDOMNode} from 'react-dom'
+import injectSheet from 'grape-web/lib/jss'
+
+import Dropdown from '../../../../dropdown/Dropdown'
+import TaskButton from './TaskButton'
+import TasksList from './TasksList'
+import CreateTask from './CreateTask'
+
+const toggleOpenState = state => ({isOpen: !state.isOpen})
+
+@injectSheet({
+  task: {
+    position: 'relative',
+    display: 'inline-block'
+  },
+  content: {
+    width: 260
+  }
+})
+export default class Task extends PureComponent {
```

Comment:
Dunno, I was looking at this component from message footer perspective where it is used. It renders the task button and on click renders the dropdown .... should the dir also be tasks?